test/compilable/minimal.d

```diff
@@ -2,7 +2,10 @@
 // POST_SCRIPT: compilable/extra-files/minimal/verify_symbols.sh
 // REQUIRED_ARGS: -c -defaultlib= runnable/extra-files/minimal/object.d
 
-// This test ensures an empty main built with a minimal runtime does not generate
-// ModuleInfo and exception handling code
+// This test ensures an empty main with a struct, built with a minimal runtime,
+// does not generate ModuleInfo or exception handling code, and does not
+// require TypeInfo
```

Comment:
Don't you have to modify `verify_symbols` to check for `TypeInfo`?

Reply:
Yeah, best to be thorough.  Done.